Gopkg.lock

```diff
@@ -35,7 +35,7 @@
   revision = "de5bf2ad457846296e2031421a34e2568e304e35"
 
 [[projects]]
-  digest = "1:35768db37daed3f5e0535a68e7dba84dc60ad2a8cedb371e4862b25fc043c243"
+  digest = "1:3d5bd56dedb305957dcefdd2381f31c43b09e12efa3722ba17f5289476677702"
```

Comment:
were new imports/packages added? does this vendor lock file need to change in this PR?  i always get concerned when I see versions of our dependencies changing

Reply:
Turns out this happened because I deleted `Gopkg.lock` and `vendor` and ran `make vendor` again to make sure everything is alright. This caused some packages to get upgraded to the latest version. I fixed it and now the only difference is 1 removed unneeded dependency!